src/CrudPanel.php

```diff
@@ -24,7 +25,7 @@
 
 class CrudPanel
 {
-    use Create, Read, Update, Delete, Errors, Reorder, Access, Columns, Fields, Query, Buttons, AutoSet, FakeFields, FakeColumns, ViewsAndRestoresRevisions, AutoFocus, Filters, Tabs, Views;
+    use Create, Read, Update, Delete, Errors, Reorder, Access, Columns, Fields, Query, Buttons, AutoSet, FakeFields, FakeColumns, ViewsAndRestoresRevisions, AutoFocus, Filters, Tabs, Boxes, Views;
```

Comment:
I know this is out of scope but the format of this `use` is just a little...unwieldy.

Personally I'd start splitting it into at least two lines :)